src/main/resources/io/sealights/plugins/sealightsjenkins/buildsteps/cli/CommandMode/PrConfigView/config.jelly

```diff
@@ -0,0 +1,36 @@
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout"
+         xmlns:t="/lib/hudson" xmlns:f="/lib/form">
+
+
+    <f:entry title="${%Technologies Options}">
+        <f:hetero-list name="techOptions" items="${instance.techOptions}" descriptors="${descriptor.technologiesDescriptors}"
+                       hasHeader="true" oneEach="true"/>
+    </f:entry>
+
+    <f:entry title="App Name" field="appName">
+        <f:textbox value="${appName}" default="$${JOB_NAME}"/>
```

Comment:
Please show that to Jason and verify that the order and caption of the fields make sense.

Reply:
What about help file that describes each field and where does the user get it?

Reply:
I'm waiting for Marc he should provide the exec git commands